localization/english/dvg_flavor_venice_l_english.yml

```diff
@@ -35,6 +170,44 @@ ig_case_fatte_per_soldi_desc:0 "Translated as Houses Made for Money, these patri
 ig_uomini_del_rinascimento:0 "Uomini del Rinascimento"
 ig_uomini_del_rinascimento_desc:0 "Meaning Reneissance Men, this group is made up of various individual patricians, scholars, and academics who advocate for a rediscovery of the Roman past and emphasis on the continued patronage of academia by the Venetian society and patrician families."
 
+#MODIFIERS
+dvg_ven_patricians:0 "Patrician Supremacy"
+dvg_ven_patricians_tt:0 "Patrician Supremacy"
+dvg_ven_radicals_victory_modifier:0 "Radical Victory"
+dvg_ven_patricians_victory_modifier:0 "Patrician Victory"
+dvg_ven_patricians_diluted_power:0 "Diluted Patrician Power"
+dvg_ven_patricians_no_outside_help:0 "Patrician Elitism"
+dvg_ven_movement_support_modifier:0 "Movement Supported"
+dvg_ven_movement_supressed_modifier:0 "Movement Suppressed"
+dvg_ven_tea_party_modifier:0 "Shipyards Sabotaged"
+dvg_ven_factory_sabotage_modifier:0 "Factories Sabotaged"
+#dvg_ven_ig_supported:0 "Interest Group Supported"
+dvg_ven_authority_up:0 "Strong Government"
+dvg_ven_authority_down:0 "Weak Government"
+dvg_ven_diverting_resources:0 "Diverting Resources"
+dvg_ven_industrialists_tax_cuts:0 "Industrialists Tax Cuts"
+dvg_ven_intelligentsia_support:0 "Intelligentsia Patronage"
+dvg_ven_modifier_public_worried:0 "Looting"
+dvg_ven_arn_diplomatic_mastry:0 "Diplomatic Mastery"
+dvg_ven_modifier_ven_payments:0 "Aragonese Payments"
+dvg_ven_modifier_arn_payments:0 "Venetian Payments"
+dvg_ven_captivated_audience:0 "Captivated Audience"
+dvg_ven_plantation_owners_support:0 "Plantation Owners Support"
+
+
+#Characters
+Giancarlo:0 "Giancarlo"
+Iannarella:0 "Iannarella"
+Felice:0 "Felice"
```

Comment:
Duplicate localization key. Key 'Felice' is defined in both 'localization/english/dvg_flavor_venice_l_english.yml' and 'localization/english/names_l_english.yml'.